docs/rhoas/rhoas-service-contexts/README.adoc

```diff
@@ -408,3 +402,260 @@ You see that the consumer component displays the stream of quote values on the w
 * https://access.redhat.com/documentation/en-us/red_hat_openshift_application_services/1/guide/8bd088a6-b7b7-4e5d-832a-b0f0494f9070#_b7f033ec-6f0c-4b3c-89b0-cb1801de19f9[CLI command reference (rhoas)^]
 * https://access.redhat.com/documentation/en-us/red_hat_openshift_streams_for_apache_kafka/1/guide/2f4bf7cf-5de2-4254-8274-6bf71673f407[ Managing account access in {product-long-kafka}^]
 * https://access.redhat.com/documentation/en-us/red_hat_openshift_service_registry/1/guide/7717db0b-9fad-4fff-91b7-b311b63290a4[Managing account access in {product-long-registry}^]
+
+[id="proc-generating-connection-information-helm_{context}"]
+== Generating connection configuration information for a Helm-based OpenShift application
+The following example shows how to use Helm to deploy an example Quarkus application on Kubernetes and then connect the Quarkus application to a Kafka instance in {product-long-kafka}. https://helm.sh/[Helm^] is an open source utility for creating, configuring, and deploying applications on Kubernetes. The Kubernetes platform referred to in the remainder of this example is Red Hat OpenShift.
+
+One part of the example Quarkus application is designed to generate random price values and produce these to a Kafka topic. Another part of the application consumes the numbers from the Kafka topic. The application uses server-sent events to expose the values as a REST UI. Finally, a web page in the application displays the exposed values.
+
+To connect the Quarkus application to a Kafka instance in {product-kafka}, you define a service context that includes the Kafka instance. You then generate connection configuration information for the service context as an OpenShift ConfigMap object and provide this to the application.
+
+.Prerequisites
+* You have a running Kafka instance in {product-kafka}.
+* You created a service context for your Kafka instance.
```

Comment:
Suggest adding perfect present tense here in line with other prereqs.

Reply:
Yes, good call.